test/support/input_parser_h264.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Membrane.VideoCompositor.Test.Pipeline.H264.TwoInputsParser do
```

Comment:
NIT: 
```suggestion
defmodule Membrane.VideoCompositor.Test.TwoInputsParser do
```

Reply:
But, it is for the testing pipeline, specifically for the H264 input.